### PR TITLE
I18n and Greek l10n of lessons and courses

### PR DESCRIPTION
**Pull request by giorgio93p**
This pull request inserts a first* Greek version of the current lessons and courses.
*there may need to be some minor changes when everything else has been translated

In order for lesson steps to continue working when modules names get translated, all lessons (also English ones, since in principle one can see an English lesson with some other interface locale) now search for module slugs/id_names instead of module names when highlighting or checking for done steps.

### CODE REVIEW

File: server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/plhthwrismos.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+</section>
+<footer>
+  <h2>Εδώ τελειώνει το μάθημα!</h2>
+  <div class="completed">
+    <p class="summary">Συγχαρητήρια! Ολοκληρώσατε την πρώτη σειρά μαθημάτων. Στη συνέχεια θα εξετάσουμε τους πόρους και θα κλείσουμε αυτή την εισαγωγή στη δημοσιογραφία δεδομένων.</p>
+  </div>
+  <div class="next">
+    <h2>Επόμενο</h2>
+    <ol>
+      <li><a href="resources">Πόροι και εμπνεύσεις για περισσότερα</a></li>
```
Comment (giorgio93p) [inline server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/plhthwrismos.html]: As in the English version of the course, this lesson (the last one of the course) links to the resources file, which is not in `index.yaml` and, hence, a 404 error is thrown. How should we treat this?

File: server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+title: Introduction to Data Journalism
+introduction_html: |-
+    <p>
```
Comment (adamhooper) [inline server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml]: Could you please translate this?

File: server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    This entry level course will teach you how to answer questions with data in a transparent
+    and reproducible manner, without the need to code.
+    </br></br>
```
Comment (adamhooper) [inline server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml]: Should be `</p><p>`. (The English one should certainly be `</p><p>`, too. I just fixed the English one. I have no patience for invalid HTML.)

File: server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml (hunk 2 above)
Comment (giorgio93p) [reply server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml]: I have already notified the translator that the file hasn't yet been translated.

**Pull request merged**